Containers/talk/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM nats:2.9.17-scratch as nats
 FROM strukturag/nextcloud-spreed-signaling:1.1.2 as signaling
-FROM coturn/coturn:4.6.2-r0-alpine
+FROM coturn/coturn:4.6.2-r3-alpine
```

Comment:
```suggestion
FROM coturn/coturn:4.6.2-alpine
```

Reply:
Maybe we could ask them if they can add the alpine version into the tag so that we can pin the alpine version?

Reply:
> I would keep that

not sure if dependabot understands that

Reply:
> Maybe we could ask them if they can add the alpine version into the tag so that we can pin the alpine version?

can you please do this?

Reply:
https://github.com/coturn/coturn/issues/1226